it/connectors/CISDeductionsConnectorISpec.scala

```diff
@@ -34,7 +37,6 @@ import scala.concurrent.Await
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration.Duration
 
-// TODO: Refactor tests. Use builders. Get rid of methods that create data and use builders instead
```

Comment:
The comment above was deleted, but the work was not done. Please address the issues before removing the TODO.